mobile/src/components/share/Header.jsx

```diff
@@ -1,11 +1,12 @@
 import React from 'react';
```

Comment:
(nit) changed the folder name share -> shared

Reply:
Changed with 7f3494c79020a437a82ddd166b92eaac45f16ef2